### PR TITLE
Mktgresp shadow bpix

### PR DESCRIPTION
**Pull request by kglotfelty**
Following from #209 and Pat's helpdesk questions, `mktgresp` now adding `BPMASK=0x03ffff` to `detsubsys`.

### CODE REVIEW

File: bin/mktgresp [hunk 1]
```diff
@@ -219,10 +219,38 @@ def make_tg_rmf( obsinfo, pha, spectrum, outroot, wvgrid_arf, wvgrid_chan):
...
+                products for HRC-I. An RMF will be created using 
+                LETG+HRC-S calibration products. This should be 
+                accurate for wavelengths < 60 A. For longer LETG+HRC-I 
+                wavelengths, the observed line widths can be increasingly 
+                broad compared with the RMF generated here owing to the 
+                flat HRC-I detector not following the geometry of the 
+                LETG Rowland torus."""
+                
+                msg = " ".join([x.strip() for x in msg.split("\n")])
+                verb0(msg)
```
Comment (DougBurke) [inline bin/mktgresp:256]: Would it make sense to use the textwrap module here? https://docs.python.org/3.6/library/textwrap.html

File: Changes.CIAO_scripts [hunk 2]
```diff
@@ -6,6 +6,9 @@
     Updated detsubsys parameter to recognize the new ACIS frame store 
     shadow bit (17).
 
+    Updated to support HRC-I+LETG LSFPARM files if they are available
+    in the CALDB.
```
Comment (DougBurke) [inline Changes.CIAO_scripts:10]: Is there a CALDB version where they were added (so we can say something like "if you have CALDB vblah or later"), or is it more complicated taht that?

File: bin/mktgresp (hunk 1 above)
Comment (kglotfelty) [reply bin/mktgresp:256]: I took a look ; text wrap is overkill here (I'd end up setting width to 999 to put all on one line). Plus it's too picky about extra spaces.

Though oddly, the latest ver on this branch does not  have this, correct `.join()` it got lost along the way when I wrapped the other warning message.  So I'll  fix that.

File: Changes.CIAO_scripts (hunk 2 above)
Comment (kglotfelty) [reply Changes.CIAO_scripts:10]: There will be (assuming Dale's release happens at same time as contrib).

File: bin/mktgresp (hunk 1 above)
Comment (DougBurke) [reply bin/mktgresp]: One of the joys of asynchronous communication, GitHub-style, is that I don't know if the `join` issue has been fixed now?

**Comment by DougBurke** [comment]
Unfortunately the changes file needs updating. Yay progress...

**Comment by kglotfelty** [comment]
this is why I don't think we should update the Changes file until we actually are ready to build the r/c.  This is a waste of time.

**Comment by kglotfelty** [comment]
I'll wait until the dax changes are merged and then I'll update for mktgresp.

**Comment by DougBurke** [comment]
On the flip side

 - the changelog and ahelp edits helps understand the PR and any future PRs you may make
 - if you don't write it now, something will get forgotten

So I think the solution is that we don't add the date to the changelog and ahelp files until the release, using an easily-greppable symbol to indicate a change is needed.

**Comment by kglotfelty** [comment]
at this point I have to start t his who pr over again (and now retest everything again) because it's also foobar'ed -- just because the changelog).

**Pull request closed without merging**